src/zkgroup/groupsend/GroupSendEndorsement.ts

```diff
@@ -53,7 +54,7 @@ export default class GroupSendEndorsement {
     return new GroupSendEndorsement(
       new Uint8Array(
         ReactNativeLibsignalClientModule.groupSendEndorsementCombine(
-          endorsements.map((next) => next.serialized)
+          endorsements.map((next) => fromByteArray(next.serialized))
```

Comment:
you can't change api with only changing ios files. is it necessary to base 64 everything?